From 8ce51859477521fe8b42e73c8e98b049dbafa644 Mon Sep 17 00:00:00 2001
From: Francois Daoust <fd@tidoust.net>
Date: Wed, 5 Mar 2025 15:17:58 +0100
Subject: [PATCH] Drop contributeToHistogramOnEvent method

The method is now defined in the base interface in the Private Aggregation API.

Pending https://github.com/WICG/turtledove/issues/1405
---
 ed/idl/turtledove.idl | 6 ------
 1 file changed, 6 deletions(-)

diff --git a/ed/idl/turtledove.idl b/ed/idl/turtledove.idl
index 64b7ee9c6..b9f50d478 100644
--- a/ed/idl/turtledove.idl
+++ b/ed/idl/turtledove.idl
@@ -220,12 +220,6 @@ dictionary PAExtendedHistogramContribution {
   bigint filteringId = 0;
 };
 
-[Exposed=InterestGroupScriptRunnerGlobalScope]
-partial interface PrivateAggregation {
-  undefined contributeToHistogramOnEvent(
-      DOMString event, PAExtendedHistogramContribution contribution);
-};
-
 [Exposed=InterestGroupBiddingAndScoringScriptRunnerGlobalScope]
 interface ForDebuggingOnly {
   undefined reportAdAuctionWin(USVString url);
-- 
2.37.1.windows.1

